addons/binding/org.openhab.binding.helios/ESH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="helios" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Helios Channel Types -->
+
+    <channel-type id="keypressedstamp">
+        <item-type>DateTime</item-type>
+        <label>Time a key was pressed</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="keypressed">
+        <item-type>String</item-type>
+        <label>Key Pressed</label>
+        <description>Code of the key pressed</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="callstate">
+        <item-type>String</item-type>
+        <label>Call State</label>
+        <description>State of the currently made call</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="calldirection" advanced="true">
+        <item-type>String</item-type>
+        <label>Call Direction</label>
+        <description>Direction of the currently made call</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="callstatestamp">
+        <item-type>DateTime</item-type>
+        <label>Time the state of the call changed</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="card">
+        <item-type>String</item-type>
+        <label>Card ID</label>
+        <description>Detected card identification with a 24-64-bit number in the hexadecimal format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="cardvalid" advanced="true">
+        <item-type>String</item-type>
+        <label>Card Valid Flag</label>
+        <description>If the detected card is valid, the parameter value is true. If not, the parameter is set to false</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="cardstamp">
+        <item-type>DateTime</item-type>
+        <label>Time the card was detected</label>
+        <description>Event date and time in CCYY-MM-DDThh:mm:ss format</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="code">
+        <item-type>String</item-type>
+        <label>Code Entered</label>
+        <description>Numerical code entered, e.g. 1234.</description>
+        <state readOnly="true" />
+    </channel-type>
+    <channel-type id="codevalid" advanced="true">
+        <item-type>String</item-type>
```

Comment:
This should be a Switch

Reply:
Btw, in ItemUtil we could add some code to convert a "false" or "true" string to ON/OFF, but I refrained from doing this. e.g. there must be a reason why this is not in there....